src/transaction/boot_cl.c

```diff
@@ -1963,6 +1964,12 @@ boot_define_class (MOP class_mop)
       return error_code;
     }
 
+  error_code = smt_add_attribute (def, "conn_server", CT_DB_SERVER_NAME, NULL);
```

Comment:
I think we need more discussion about adding that column.

We would appreciate it if we could discuss it through another PR after sufficient discussion about whether it is necessary to modify _db_class and db_class while adding _db_server and db_server.

Reply:
Yes, we will cancel the modification of the _db_class catalog and decide whether to reflect it after further discussion.